src/OpenLoco/Graphics/Gfx.h

```diff
@@ -88,6 +88,16 @@ namespace OpenLoco::Gfx
     };
 
 #pragma pack(pop)
+    namespace G1ElementFlags
+    {
+        constexpr uint16_t hasTransparancy = 1 << 0; // Image data contains transparent sections (when not set data is plain bmp)
+        constexpr uint16_t unk1 = 1 << 1;            // Unknown function not used on any entry
```

Comment:
Is this true for RCT2 as well as Locomotion?

Reply:
Yeah I've never found a sprite that has this flag set in either rct2 or locomotion